sli/name.cc

```diff
@@ -124,7 +124,8 @@ Name::info( std::ostream& out )
 }
 
 
-std::ostream& operator<<( std::ostream& o, const Name& n )
+std::ostream&
+operator<<( std::ostream& o, const Name& n )
```

Comment:
Strangely here it works, in `streamers.cpp` it didn't... :thinking:

Reply:
Let's not be too confused, but happy that it works here. Another mystery solved by deleting the code containing it!